Sentry.Xamarin.Forms/Extensions/SentryOptionsExtensions.cs

```diff
@@ -41,15 +41,15 @@ public static void DisableXamarinFormsCache(this SentryOptions options)
         }
 
         internal static SentryXamarinOptions GetSentryXamarinOptions(this SentryOptions options)
-            => SentryXamarin.Options;
+            => options is SentryXamarinOptions xamarinOptions ? xamarinOptions : SentryXamarin.Options;
```

Comment:
This is super hacky. Depending on the type you return it otherwise some static instance?
Sounds like something fundamentally broken in the design.
Why is there a static one?

Reply:
for some reason, I was expecting the user to be able to interact with SentryOptions after Init was called, like inside of SentrySdk.WithScope...